__tests__/app.test.js

```diff
@@ -28,4 +28,17 @@ describe('/api/topics',()=>{
             expect(message).toBe('Route not found')
         })
     })
+})
+
+describe('/api',()=>{
+    test('this end point should respond with a list of all other endpoints available',()=>{
```

Comment:
I can only see changes in your app.test.js file and no other files. Can you make sure you commit any files you've changed? I also think you may have misunderstood this one. There is a file called endpoints.json where you can add documentation for each of your endpoints as you create them. Then you can just require this in and send it back to the user.